bundle/language_it.properties

```diff
@@ -492,6 +492,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = Positivo
 angal.lab.printreport                                                            = Stampa rapporto
 angal.lab.printtable                                                             = Stampa tabella
+angal.lab.print																	 = Stampa
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d